gpiozero/boards.py

```diff
@@ -1324,6 +1324,101 @@ def __init__(self, *, pwm=False, initial_value=False, pin_factory=None):
         )
 
 
+class ModMyPiStar(LEDBoard):
+    """
+    Extends :class:`LEDBoard` for the `ModMyPi Programmable Christmas Tree Star`_:
+    a Christmas tree star with 30 white LEDs.
+
+    The 30 white LEDs can be accessed through the attributes inner and outer.
+    The inner attribute controls the five inner LEDs, while the outer attribute
+    controls the 25 outer LEDs. Alternatively, as with all descendents of
+    :class:`LEDBoard`, you can treat the instance as a sequence of LEDs (the
+    first element is the inner LEDs).
+
+    The ModMyPi star board pins are fixed and therefore there's no need to
+    specify them when constructing this class. The following example turns on
+    all the inner LEDs before turning on the outer LEDs one at a time::
+
+        from gpiozero import ModMyPiStar
+        from time import sleep
+
+        tree = ModMyPiStar()
+
+        for light in tree:
+            light.on()
+            sleep(1)
+
+    The following example turns on the outer LEDs successively and increasingly
+    quickly before flashing the inner LEDs::
+
+        from time import sleep
+        from star import ModMyPiStar
+
+        step = 0.5
+        count = 0
+        star = ModMyPiStar(pwm=True)
+        leds = star.leds
+
+        while True:
+            if(count%26!=0):
+                leds[count%26].on()
+                sleep(step)
+                leds[count%26].off()
+
+            count += 1
+            step = step*0.99
+
+            if(step <= 0.0001):
```

Comment:
```suggestion
            if step <= 0.0001:
```
Brackets aren't needed for simple `if` expressions.